docs/sources/setup-grafana/configure-docker.md

```diff
@@ -60,48 +93,262 @@ The following settings are hard-coded when launching the Grafana Docker containe
+You can apply this technique to any configuration options in `conf/grafana.ini` by setting `GF_<SectionName>_<KeyName>__FILE` to the file path that contains the secret information. For more information about Docker secret command usage, refer to [docker secret](https://docs.docker.com/engine/reference/commandline/secret/).
 
-You can do this with any of the configuration options in conf/grafana.ini by setting `GF_<SectionName>_<KeyName>__FILE` to the path of the file holding the secret.
-
-For example, you could set the admin password this way:
+The following example demonstrates how to set the admin password:
 
 - Admin password secret: `/run/secrets/admin_password`
 - Environment variable: `GF_SECURITY_ADMIN_PASSWORD__FILE=/run/secrets/admin_password`
 
-## Configure AWS credentials for CloudWatch Support
+### Configure Docker secrets credentials for AWS CloudWatch
+
+Grafana ships with built-in support for the [Amazon CloudWatch datasource]({{< relref "../datasources/aws-cloudwatch" >}}). To configure the data source, you must provide information such as the AWS ID-Key, secret access key, region, and so on. You can use Docker secrets as a way to provide this information.
+
+Example:
+
+The example below shows how to use Grafana environment variables via Docker Secrets for the AWS ID-Key, secret access key, region, and profile.
+
+The example uses the following values for the AWS Cloudwatch data source:
 
 ```bash
-docker run -d \
--p 3000:3000 \
---name=grafana \
--e "GF_AWS_PROFILES=default" \
--e "GF_AWS_default_ACCESS_KEY_ID=YOUR_ACCESS_KEY" \
--e "GF_AWS_default_SECRET_ACCESS_KEY=YOUR_SECRET_KEY" \
--e "GF_AWS_default_REGION=us-east-1" \
-grafana/grafana-enterprise
+AWS_default_ACCESS_KEY_ID=aws01us02
+AWS_default_SECRET_ACCESS_KEY=topsecret9b78c6
+AWS_default_REGION=us-east-1
 ```
 
-You may also specify multiple profiles to `GF_AWS_PROFILES` (e.g.
-`GF_AWS_PROFILES=default another`).
+1. Create a Docker secret for each of the values noted above.
+
+   ```bash
+   echo "aws01us02" | docker secret create aws_access_key_id -
+   ```
+
+   ```bash
+   echo "topsecret9b78c6" | docker secret create aws_secret_access_key -
+   ```
+
+   ```bash
+   echo "us-east-1" | docker secret create aws_region -
+   ```
+
+1. Run the following command to determine that the secrets were created.
 
-Supported variables:
+   ```bash
+   $ docker secret ls
+   ```
+
+   The output from the command should look similar to the following:
+
+   ```
+   ID                          NAME           DRIVER    CREATED              UPDATED
+   i4g62kyuy80lnti5d05oqzgwh   aws_access_key_id             5 minutes ago        5 minutes ago
+   uegit5plcwodp57fxbqbnke7h   aws_secret_access_key         3 minutes ago        3 minutes ago
+   fxbqbnke7hplcwodp57fuegit   aws_region                    About a minute ago   About a minute ago
+   ```
+
+   Where:
+
+   ID = the secret unique ID which we will use in the docker run command
+
+   NAME = the we defined for each secret
```

Comment:
@usmangt , not sure what this means. let's not use the word 'we'. Could this be "That you defined for each secret"?

Reply:
fixed now!!